### DISCUSSION TITLE
webpack re-builds unstable, sometimes breaks the app

### DISCUSSION THREAD

**Discussion by tu4mo** [webpack/webpack#13631] [General]
# Bug report

**What is the current behavior?**

I have a simple webpack React app that has a dependency of a large-ish component library. webpack (running with `serve` in `development` mode) notices when I re-build the library but sometimes the app will break and I have to save a file for webpack to start the re-build again and then it might work. Sometimes the re-build process starts multiple times.

Is there anything I could do to make webpack's re-build process more stable when the library is updated (= multiple files get's recreated in a short timeframe)?

**If the current behavior is a bug, please provide the steps to reproduce.**

Use a dependecy of a large library with many files that get's re-created.

**What is the expected behavior?**

Re-builds are fast and the app works everytime.

**Other relevant information:**
webpack version:  5.40.0
Node.js version: 16.4.0
Operating System: macOS v11.4

**Comment by alexander-akait** [comment]
Please provide reproducible example
